Documentation/gettinghelp.rst

```diff
@@ -4,18 +4,23 @@
 Getting Help
 ############
 
-We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions (FAQ)`_.
-Check there to see if your question(s) is already addressed.
+Cilium is a project  with a growing community, so if you need help there a
+numerous ways to getting help:
 
-The best way to get help if you get stuck is to contact us on the
-`Cilium Slack channel <https://cilium.herokuapp.com>`_.
+**Cilium Frequently Asked Questions (FAQ)**: Cilium uses `GitHub
+tags <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20>`_
+to maintain a list of users question. So we suggest you to check if your
+question is already addressed.
```

Comment:
"So we suggest you to check if your question is already addressed." -> "We suggest that you check if your question is already addressed." (I dropped the "So" and add a "that".

Reply:
to main a list of questions asked by users. We suggest checking to see if your question is already answered.